examples/openvino/question-answering/README.md

```diff
@@ -44,4 +44,32 @@ python run_qa.py \
   --max_seq_length 384 \
   --doc_stride 128 \
   --output_dir /tmp/outputs_squad/
-```
\ No newline at end of file
+```
+
+### Joint Pruning, Quantization and Distillation (JPQD) for BERT on SQuAD1.0
+`OVTrainer` also provides an advanced optimization workflow through the NNCF when Transformer model can be structurally pruned along with 8-bit quantization and distillation. Below is an example which demonstrates how to jointly prune, quantize BERT-base for SQuAD 1.0 using NNCF config `--nncf_compression_config` and distill from BERT-large teacher. This example closely resembles the movement sparsification work of [Lagunas et al., 2021, Block Pruning For Faster Transformers](https://arxiv.org/pdf/2109.04838.pdf). This example takes about 12 hours with a single V100 GPU and the output model attains about 40% structured sparsity (denominated by linear layers only). The joint pruned and quantized OpenVINO IR provides >60% more throughput over quantize-only model on AWS EC2 instance (c6i.32xlarge). Attributed to large teacher distillation, the sparse-quantized model has F1 improved to >89%.
+
+More on how to configure movement sparsity, see NNCF documentation [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/torch/sparsity/movement/MovementSparsity.md).
+```bash
+python run_qa.py \
+    --model_name_or_path bert-base-uncased \
+    --dataset_name squad \
+    --teacher_model_or_path bert-large-uncased-whole-word-masking-finetuned-squad \
+    --distillation_weight 0.9 \
+    --do_eval \
+    --fp16 \
+    --do_train \
+    --learning_rate 3e-5 \
+    --num_train_epochs 8 \
+    --per_device_eval_batch_size 128 \
+    --per_device_train_batch_size 16 \
+    --max_seq_length 384 \
+    --doc_stride 128 \
+    --logging_steps 1 \
+    --evaluation_strategy steps \
+    --eval_steps 250 \
+    --save_steps 500 \
```

Comment:
Do we need to save every 500 steps by default? This saved about 180GB of checkpoints on my disk. Not everyone has a very large harddisk, it's annoying when you set a demo to train overnight and wake up to a completely full harddisk and training that crashed half way through because of that.

Reply:
Agree, I found it annoying too when I started to use HF's transformers. Added checkpoint limit in https://github.com/huggingface/optimum-intel/pull/150/commits/3e7ff7879d15eb5a02f4a099fa9a9e0e196abcc9 .